src/test/java/com/google/api/codegen/testdata/discoveries/ruby/ruby_genomics.v1.json.baseline

```diff
@@ -0,0 +1,1801 @@
+
+service.authorization = \
+    Google::Auth.get_application_default(['https://www.googleapis.com/auth/cloud-platform'])
+
+# TODO: Assign values to desired members of `request_body`:
+request_body = Google::Apis::GenomicsV1::SearchReadsRequest.new
+
+alignments = service.fetch_all(items: :alignments) do |token|
+  service.search_reads(request_body, page_token: token)
+end
+
+alignments.each do |read|
+  # TODO: Change code below to process each `read` resource:
+  puts JSON.pretty_generate(read)
+end
+# BEFORE RUNNING:
+# ---------------
+# 1. If not already done, enable the Genomics API
+#    and check the quota for your project at
+#    https://console.developers.google.com/apis/api/genomics
+# 2. This sample uses Application Default Credentials for authentication.
+#    If not already done, install the gcloud CLI from
+#    https://cloud.google.com/sdk and run
+#    `gcloud beta auth application-default login`
+#    For more information, see:
+#    https://developers.google.com/identity/protocols/application-default-credentials
+# 3. Install the Ruby client library and Application Default Credentials
+#    library by running `gem install google-api-client` and
+#    `gem install googleauth`
+
+require 'googleauth'
+require 'google/apis/genomics_v1'
+require 'json'
+
+service = Google::Apis::GenomicsV1::GenomicsService.new
+
+service.authorization = \
+    Google::Auth.get_application_default(['https://www.googleapis.com/auth/cloud-platform'])
+
+# TODO: Assign values to desired members of `request_body`:
+request_body = Google::Apis::GenomicsV1::StreamReadsRequest.new
+
+response = service.stream_reads(request_body)
+
+# TODO: Change code below to process the `response` object:
+puts JSON.pretty_generate(response)
+# BEFORE RUNNING:
+# ---------------
+# 1. If not already done, enable the Genomics API
+#    and check the quota for your project at
+#    https://console.developers.google.com/apis/api/genomics
+# 2. This sample uses Application Default Credentials for authentication.
+#    If not already done, install the gcloud CLI from
+#    https://cloud.google.com/sdk and run
+#    `gcloud beta auth application-default login`
+#    For more information, see:
+#    https://developers.google.com/identity/protocols/application-default-credentials
+# 3. Install the Ruby client library and Application Default Credentials
+#    library by running `gem install google-api-client` and
+#    `gem install googleauth`
+
+require 'googleauth'
+require 'google/apis/genomics_v1'
+require 'json'
+
+service = Google::Apis::GenomicsV1::GenomicsService.new
+
+service.authorization = \
+    Google::Auth.get_application_default(['https://www.googleapis.com/auth/cloud-platform'])
+
+# The ID of the reference.
+reference_id = ''  # TODO: Update placeholder value.
+
+sequence = service.fetch_all(items: :sequence) do |token|
+  service.list_reference_bases(reference_id, page_token: token)
+end
+
+sequence.each do |item|
```

Comment:
Ruby is special because of the `fetch_all` method - I actually reported this as a bug to the generator a few months ago and it was fixed, so I believe it's being handled as intended.